roles/common/install_dependencies/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Install dependencies
+  apt:
+    name: "{{ item }}"
+    state: present
+    update_cache: yes
+  with_items:
+     "{{ packages }}"
```

Comment:
The variable name `packages` isn't entirely clear what it's used for. Suggest something like `required_packages` or `package_dependencies` or a better idea than those suggestions to clarify what the variable is for.

Reply:
`required_packages`

Reply:
done